app/controllers/alchemy/admin/tags_controller.rb

```diff
@@ -4,9 +4,12 @@ class TagsController < ResourcesController
       before_filter :load_tag, only: [:edit, :update, :destroy]
 
       def index
-        @tags = ActsAsTaggableOn::Tag.where(
-          "name LIKE '%#{params[:query]}%'"
-        ).page(params[:page] || 1).per(per_page_value_for_screen_size).order("name ASC")
+        @query = ActsAsTaggableOn::Tag.ransack(params[:q])
+        @tags = @query
+                  .result
+                  .page(params[:page] || 1)
```

Comment:
Style/MultilineOperationIndentation: Use 2 (not 10) spaces for indenting an expression in an assignment spanning multiple lines.